Vagrantfile

```diff
@@ -193,7 +193,15 @@ def provision(config,
   raise ArgumentError.new('update_command is required') if update_command == 'required'
   raise ArgumentError.new('update_tracking_file is required') if update_tracking_file == 'required'
   raise ArgumentError.new('install_command is required') if install_command == 'required'
-  config.vm.provision "bats dependencies", type: "shell", inline: <<-SHELL
+  config.vm.provider "virtualbox" do |v|
+    # Give the box more memory and cpu because our tests are beasts!
+    v.memory = Integer(ENV['VAGRANT_MEMORY'] || 8192)
+    v.cpus = Integer(ENV['VAGRANT_CPUS'] || 4)
+  end
+  config.vm.synced_folder "#{Dir.home}/.vagrant/gradle/caches", "/home/vagrant/.gradle/caches",
```

Comment:
I do not doubt the correctness of this, but this is a bad restriction. This is getting off-topic, but what is the solution on a Jenkins server that is running multiple jobs in parallel (so all would be sharing the same cache in `/var/lib/jenkins/.gradle`)? Is this not feasible?